controllers/hivelocitymachine_controller.go

```diff
@@ -159,7 +159,8 @@ func (r *HivelocityMachineReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	return r.reconcileNormal(ctx, machineScope)
 }
 
-func breakReconcile(ctrl *reconcile.Result, err error) (reconcile.Result, bool, error) { // why name "ctrl"? This is already the package "sigs.k8s.io/controller-runtime"
+func breakReconcile(ctrl *reconcile.Result, err error) (reconcile.Result, bool, error) { // question: why name "ctrl"? This is already the package "sigs.k8s.io/controller-runtime"
+	// question: it is hard to understand what this function does. It gets used twice. Maybe inline it?
```

Comment:
I'll think about an alternative. "ctrl" comes from the second way of importing "Result". If used with reconcile.Result it does not make much sense...